### DISCUSSION TITLE
Provide a possibility to delete service persisted data on service delete

### DISCUSSION THREAD

**Discussion by oesukam** [coollabsio/coolify#1352] [Feature Requests]
I've observed that when services are deleted, their associated data remains on the disk, continuing to occupy storage space.

Users can be given the option to decide whether they want to completely remove all data related to the service upon deletion.

**Comment by peaklabs-dev** [comment]
@oesukam Will be fixed via: https://github.com/coollabsio/coolify/pull/3024

**Comment by peaklabs-dev** [comment]
Will be available in the next version
